pallets/did/src/tests.rs

```diff
@@ -225,19 +230,22 @@ fn check_successful_complete_update() {
 	let old_att_key = get_ed25519_attestation_key(true);
 	let new_att_key = get_ed25519_attestation_key(false);
 	let new_del_key = get_sr25519_attestation_key(true);
-	let new_url: UrlEncoding = "https://new_kilt.io".into();
+	let new_url = did::Url::from(
+		did::HttpUrl::try_from("https://new_kilt.io".as_bytes())
+			.expect("https://new_kilt.io should not be considered an invalid HTTP URL."),
+	);
 
 	let mut old_did_details =
 		generate_mock_did_details(did::PublicVerificationKey::from(old_auth_key.public()), old_enc_key);
-	old_did_details.attestation_key = Some(PublicVerificationKey::from(old_att_key.public()));
+	old_did_details.attestation_key = Some(did::PublicVerificationKey::from(old_att_key.public()));
```

Comment:
Is there any reason you removed `PublicVerificationKey` from the imports?

Reply:
No. It was just a matter of consistency, as I was accessing the did pallet using the full path, and in some places I was using `PublicVerificationKey` after importing it directly.